src/reader/buffered_reader.rs

```diff
@@ -393,6 +397,26 @@ impl<R: BufRead> Reader<R> {
             buf.clear();
         }))
     }
+
+    /// Same as `read_to_end_into` but returns all the bytes read instead of the indices.
+    pub fn read_text_into<'b>(
+        &mut self,
+        end: QName,
+        buf: &mut Vec<u8>,
+        output_buf: &'b mut Vec<u8>,
+    ) -> Result<&'b str> {
```

Comment:
I would like to do without a temporary buffer, at least in the API, at best at all. Of course, that means that `read_to_end_into` cannot be used directly...